ghost/core/test/e2e-api/admin/__snapshots__/emails.test.js.snap

```diff
@@ -753,7 +753,7 @@ Object {
       "reply_to": null,
       "source": null,
       "source_type": "html",
-      "status": "pending",
+      "status": "failed",
```

Comment:
Need to check if this is a problem in the frontend, but is expected behaviour in the backend because the change of status happens in a job

Reply:
We don't use this endpoint in admin + we poll after retrying